charts/rekor-tiles/rekor-tiles/templates/_helpers.tpl

```diff
@@ -0,0 +1,182 @@
+- "rekor-server"
+- "serve"
+{{- if .Values.antispam }}
+{{- if .Values.antispam.persistent }}
+- "--persistent-antispam"
+{{- end }}
+{{- if .Values.antispam.maxBatchSize }}
+- {{ printf "--antispam-max-batch-size=%d" (.Values.antispam.maxBatchSize | int) | quote }}
+{{- end }}
+{{- if .Values.antispam.pushbackThreshold }}
+- {{ printf "--antispam-pushback-threshold=%d" (.Values.antispam.pushbackThreshold | int) | quote }}
+{{- end }}
+{{- end }}
+{{- if .Values.tesseraLifecycle }}
+{{- if .Values.tesseraLifecycle.batchMaxAge }}
+- {{ printf "--batch-max-age=%s" .Values.tesseraLifecycle.batchMaxAge | quote }}
+{{- end }}
+{{- if .Values.tesseraLifecycle.batchMaxSize }}
+- {{ printf "--batch-max-size=%d" (.Values.tesseraLifecycle.batchMaxSize | int) | quote }}
+{{- end }}
+{{- if .Values.tesseraLifecycle.checkpointInterval }}
+- {{ printf "--checkpoint-interval=%s" .Values.tesseraLifecycle.checkpointInterval | quote }}
+{{- end }}
+{{- if .Values.tesseraLifecycle.pushbackMaxOutstanding }}
+- {{ printf "--pushback-max-outstanding=%d" (.Values.tesseraLifecycle.pushbackMaxOutstanding | int) | quote }}
+{{- end }}
+{{- end }}
+{{- if .Values.clientSigningAlgorithms }}
+- {{ printf "--client-signing-algorithms=%s" .Values.clientSigningAlgorithms | quote }}
+{{- end }}
+- {{ printf "--gcp-bucket=%s" .Values.gcp.bucket | quote }}
+- {{ printf "--gcp-spanner=%s" .Values.gcp.spanner | quote }}
+- "--grpc-address=0.0.0.0"
+{{- if .Values.grpc }}
+{{- if .Values.grpc.port }}
+- {{ printf "--grpc-port=%d" (.Values.grpc.port | int) | quote }}
+{{- end }}
+{{- end }}
+{{- if .Values.hostname }}
+- {{ printf "--hostname=%s" .Values.hostname | quote }}
+{{- end }}
+- "--http-address=0.0.0.0"
+{{- if .Values.http }}
+{{- if .Values.http.metricsPort }}
+- {{ printf "--http-metrics-port=%d" (.Values.http.metricsPort | int) | quote }}
+{{- end }}
+{{- if .Values.http.port }}
+- {{ printf "--http-port=%d" (.Values.http.port | int) | quote }}
+{{- end }}
+{{- end }}
+{{- if .Values.server }}
+{{- if .Values.server.maxRequestBodySize }}
+- {{ printf "--max-request-body-size=%d" (.Values.server.maxRequestBodySize | int) | quote }}
+{{- end }}
+{{- if .Values.server.timeout }}
+- {{ printf "--timeout=%s" .Values.server.timeout | quote }}
+{{- end }}
+{{- end }}
+{{- if .Values.readOnly }}
+- "--read-only"
+{{- end }}
+{{- if .Values.signer }}
+{{- if (.Values.signer.file).path }}
+- {{ printf "--signer-filepath=%s" .Values.signer.file.path | quote }}
+{{- end }}
+{{- if (.Values.signer.file.secret).name }}
+- "--signer-password=${PASSWORD}"
+{{- end }}
+{{- end }}
+{{- if .Values.signer.kms }}
+{{- if .Values.signer.kms.hash }}
+- {{ printf "--signer-kmshash=%s" .Values.signer.kms.hash | quote }}
+{{- end }}
+- {{ printf "--signer-kmskey=%s" .Values.signer.kms.key | quote }}
+{{- if .Values.signer.tink }}
+- {{ printf "--signer-tink-kek-uri=%s" .Values.signer.tink.kekURI | quote }}
+- "--signer-tink-keyset-path=/etc/tink-config/tink.keyset.enc
```

Comment:
it would be helpful to also be able to have a caller specify `extraArgs` which get appended without having to update the helm chart

Reply:
done